packages/components/src/components/notification/notification.tsx

```diff
@@ -91,8 +93,13 @@ export class Notification {
 
   connectedCallback() {
     if (this.hostElement.hasAttribute('opened')) {
-      // Do not use `role="alert"` if opened/visible on page load
-      this.role = undefined;
+      if (
+        this.innerAriaLive === 'polite' ||
+        this.innerRole === undefined ||
```

Comment:
Is it even possible that `this.innerRole` is `undefined` here? It has a default value of `'alert'`. Maybe only set it to `'status'` if `this.innerAriaLive === 'polite'`.

Otherwise, if `this.innerRole` can actually be `undefined` here, I'd suggest that this should be treated like the default, i.e. `'alert'` would be used.